File: desafio_tatico/desafio_tatico.py
P = int(input()) #Número de jogadores
S = int(input()) #Quantidade de soldados
L = P*S
i=0
k=0
resA=0
resD=0
j=0
ATK = []
DEF = []
while L>i:
    i+=1
    A,D= map(int,input().split())
    ATK.append(A)
    DEF.append(D)

while P>j:
    j+=1
    while S>k:
        resA += ATK[k]
        resD += DEF[k]
        k+=1
    print(f'{resA} {resD}')
    resA = 0
    resD = 0
    S+=(P+1)

#3 Blocos de 4 Linhas
#Bloco = P
#Linhas = S

File: desafio_tatico/desafio_tatico.cpp
#include <iostream>
#include <vector>
using namespace std;

int main(){
    vector<int> ATK;
    vector<int> DEF;
    int P,S,L=0,resA,resB,A,D,k=0;
    cin >> P;
    cin >> S;
    L = P*S;

    for(int i=0;L>i;i++){
        cin >> A >> D;
        ATK.push_back(A);
        DEF.push_back(D);
    }

    for(int i=0;P>i;i++){
        resA = 0;
        resB = 0;
        while (S>k){
            resA += ATK[k];
            resB += DEF[k];
            k++;
        }
        cout << resA << " " << resB << endl;
        S+=(P+1);
    }

}